### PR TITLE
DDF-3516 Converted coordinates to display in OpenLayers

### PR DESCRIPTION
**Pull request by millerw8**
Converted coordinates from map projection to EPSG:4326 to display lat/lng to use

#### Who is reviewing it? 
@Variadicism 
@mackncheesiest 

(please choose AT LEAST two reviewers that need to approve the PR before it can get merged)
#### Select relevant component teams: 
https://github.com/orgs/codice/teams
#### Choose 2 committers to review/merge the PR. 
(please choose ONLY two committers from below, delete the rest)
@andrewkfiedler
@bdeining
#### How should this be tested? (List steps with links to updated documentation)
Try out the UI when the projection is set to EPSG:4326, confirm the 2D map shows correct lat/lng values in the lower left as the user moves around the cursor.  Repeat for EPSG:3857

#### What are the relevant tickets?
[DDF-3516](https://codice.atlassian.net/browse/DDF-3516)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: catalog/ui/catalog-ui-search/src/main/webapp/component/visualization/maps/openlayers/map.openlayers.js [hunk 1]
```diff
@@ -89,9 +89,10 @@ module.exports = function OpenlayersMap(insertionElement, selectionInterface, no
   
     function setupTooltip(map) {        
         map.on('pointermove', function(e){
+            var point = unconvertPointCoordinate(e.coordinate);
```
Comment (mackncheesiest) [inline catalog/ui/catalog-ui-search/src/main/webapp/component/visualization/maps/openlayers/map.openlayers.js]: This doesn't cause any unintended side effects with non EPSG:3857 maps?

**Comment by bdeining** [comment]
build now

**Comment by millerw8** [comment]
build now

**Comment by bdeining** [comment]
hero successful; verified coordinates on the 2D map in EPSG:4326 and EPSG:3587

**Pull request merged**